tests/scripts/test_config.sh

```diff
@@ -22,3 +31,16 @@ expect_fail pulp --username test --password test --client "/some/path" status
 
 # fail when using basic auth and cert auth
 expect_fail pulp --key "/some/path" file remote list
+
+
+# CONFIG COMMAND
+
+expect_fail pulp config edit --location $test_settings
+
+expect_succ pulp config create --location $test_settings --base-url "http://testing"
+cat $test_settings | grep 'base_url = "http://testing"'
+cat $test_settings | grep 'verify_ssl = true'
+
+expect_fail pulp config create --location $test_settings
```

Comment:
`make me a sandwich`
`access denied`
`sudo make me a sandwich`

```suggestion
expect_fail pulp config create --location $test_settings
expect_succ pulp config create --location $test_settings --overwrite
```